event.c

```diff
@@ -350,8 +350,13 @@ static void event_image_snapshot(struct context *cnt, int type ATTRIBUTE_UNUSED,
             void *dummy2 ATTRIBUTE_UNUSED, struct tm *currenttime_tm)
 {
     char fullfilename[PATH_MAX];
-
-    if (strcmp(cnt->conf.snappath, "lastsnap")) {
+    int len = strlen(cnt->conf.snappath);
+    int offset = 0;
+    if (len >= 9)
```

Comment:
no problem, offset would be zero, and line 359 would not be equal to
"lastsnap".

My check of >= to 9 is just so I don't create a negative offset.


*Jim Dodgen*







On Sat, May 16, 2015 at 1:30 PM, tosiara <notifications@github.com> wrote:

> In event.c
> <https://github.com/Mr-Dave/motion/pull/73#discussion_r30464342>:
>
> > @@ -350,8 +350,13 @@ static void event_image_snapshot(struct context *cnt, int type ATTRIBUTE_UNUSED,
> >              void *dummy2 ATTRIBUTE_UNUSED, struct tm *currenttime_tm)
> >  {
> >      char fullfilename[PATH_MAX];
> > -
> > -    if (strcmp(cnt->conf.snappath, "lastsnap")) {
> > +    int len = strlen(cnt->conf.snappath);
> > +    int offset = 0;
> > +    if (len >= 9)
>
> What if I use string "12345678"?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/Mr-Dave/motion/pull/73/files#r30464342>.
>